js/test/components/operatorSignIn/attestation.test.tsx

```diff
@@ -77,7 +80,29 @@ describe("Attestation", () => {
           employees={EMPLOYEES}
         />,
       );
-      expect(view.getByRole("textbox")).toHaveValue("123");
+      const input = view.getByLabelText(/Operator Badge Number/i, {
+        selector: "input",
+      });
+      expect(input).toHaveValue("123");
+    });
+  });
+
+  describe("radio text box", () => {
```

Comment:
Is there any behavior for the radio box that needs to be tested besides the happy path? Maybe that you can't submit the attestation if it's blank?

Reply:
oop, thanks! fixed